recipe/meta.yaml

```
@@ -56,6 +60,31 @@ outputs:
         convenient.
 
         This package contains utility modules to talk to Avahi with Python easier.
+  - name: avahi
+    script: lib-avahi.sh
+    requirements:
+      build:
+        # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+        # Compilers are named 'c', 'cxx' and 'fortran'.
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - make
+        - pkg-config
+        - intltool
+        - make
+      host:
+        - libevent
+        - libdaemon
+        - dbus >=0.34
+    test:
+      requires:
+        - dbus >=0.34
+        - libdaemon
+        - libevent
```

These are presumably actually the runtime requirements for the library so should be specified as `requirements/run` and not `test/requires:

```suggestion
      run:
        - dbus >=0.34
        - libdaemon
        - libevent
    test:
```